### ISSUE TITLE
Feature Request:  .NET Core and Mac support

### ISSUE THREAD

**Issue by johnattrender** [MoistCoder/CockyGrabber#38]
Is it possible to convert this to .NET Core and make it run on a Mac?  Thoughts?

**Comment by MoistCoder** [comment]
Yes as it's stated in the Readme file: Cockygrabber will eventually migrate to .NET (Core). This migration will take some time as I will need to find replacements for some pieces of code which take use of the .NET framework such as libraries. On top of that development-time there is the addition of my social life, so it's going to take a while for the transition to a frameworkless project.

**Comment by johnattrender** [comment]
Hey thanks for the quick response!  I missed that in the Readme (my fault).  Totally understand the social life;  I was just wondering.  Regardless, thank you for your work.  Really awesome job.

**Comment by MoistCoder** [comment]
Thank you
